docs/other/090_pod_scheduling.adoc

```diff
@@ -22,7 +22,7 @@ The following settings can be configured:
 
 |`nodeSelector`
 |`nodeSelector` is the simplest recommended form of node selection constraint.
-`nodeSelector` is a field of role spec, it specifies a map of key-value pairs.
+`nodeSelector` is a field of role spec, it specifies an array of key-value pairs.
```

Comment:
This is not correct.
The documentation here is correct, node selector is a map of key value pairs. It is the example yaml below that is wrong.
```
  nodeSelector:
    - disktype: ssd
```
should be
```
  nodeSelector:
    disktype: ssd
```